netci.groovy

```diff
@@ -109,53 +109,6 @@ def addPushJob(String project, String branch, String os, String configuration)
   };
 };
 
-// Tarball builds that are not enforced to be offline
-[true, false].each { isPR ->
-  ["RHEL7.2", "CentOS7.1"].each { os ->
-    ["Release", "Debug"].each { configuration ->
-
-      def shortJobName = "${os}_Tarball_${configuration}";
-      def contextString = "${os} Tarball ${configuration}";
-      def triggerPhrase = "(?i).*test\\W+${contextString}.*";
-
-      def newJob = job(Utilities.getFullJobName(project, shortJobName, isPR)){
-        steps{
-            shell("cd ./source-build;git submodule update --init --recursive");
-            shell("cd ./source-build;./build.sh /p:ArchiveDownloadedPackages=true /p:Configuration=${configuration} ${loggingOptions}");
-            shell("cd ./source-build;./build-source-tarball.sh ../tarball-output --skip-build");
-
-            shell("cd ./tarball-output;./build.sh /p:Configuration=${configuration} ${loggingOptions}")
-            shell("cd ./tarball-output;./smoke-test.sh --minimal --configuration ${configuration}")
-        }
-      }
-
-      setMachineAffinity(newJob, os);
-
-      Utilities.standardJobSetup(newJob, project, isPR, "*/${branch}");
-
-      // Increase timeout. The tarball builds can take longer than the 2 hour default.
-      Utilities.setJobTimeout(newJob, 240);
-
-      // Clone into the source-build directory
-      Utilities.addScmInSubDirectory(newJob, project, isPR, 'source-build');
-
-      addArchival(newJob);
-      if(isPR){
-        if(configuration == "Release"){
-          Utilities.addGithubPRTriggerForBranch(newJob, branch, contextString);
-        }
-        else{
-          Utilities.addGithubPRTriggerForBranch(newJob, branch, contextString, triggerPhrase);
-        }
-      }
-      else{
-        Utilities.addGithubPushTrigger(newJob);
-      }
-
-    }
-  }
-}
-
 // Tarball builds that are enforced offline with unshare
 [true, false].each { isPR ->
   ["RHEL7.2"].each { os->
```

Comment:
Could you add CentOS here? I forgot that we only had RHEL7.2 here, and I think we should try to keep making and building the tarball on both.

Reply:
Sure, I will try it out with the CentOS docker image.